src/test/java/org/openmrs/contrib/qaframework/automation/MarkPatientDeceasedSteps.java

```diff
@@ -0,0 +1,71 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import static org.junit.Assert.assertTrue;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.helper.TestData;
+import org.openmrs.contrib.qaframework.page.FindPatientPage;
+import org.openmrs.contrib.qaframework.page.MarkPatientDeceasedPage;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+public class MarkPatientDeceasedSteps extends Steps {
+
+	MarkPatientDeceasedPage markPatientDeceasedPage = new MarkPatientDeceasedPage(page);
+
+	private TestData.PatientInfo patient;
+
+	@Before(RunTest.HOOK.MARK_PATIENT_DECEASED)
```

Comment:
After updating your RunTest class, use 
```suggestion
	@Before(RunTest.HOOK.SELENIUM_MARK_PATIENT_DECEASED)
```